src/views/darkroom.c

```diff
@@ -421,12 +421,11 @@ void expose(
     if(dev->image_invalid_cnt)
     {
       fontsize = DT_PIXEL_APPLY_DPI(20);
-      load_txt = dt_util_dstrcat(NULL, "%s `%s', %s\n\n%s\n%s",
-          _("darktable could not load"),
-          dev->image_storage.filename,
+      load_txt = dt_util_dstrcat(
+          NULL, NC_("darkroom", "%s `%s', %s\n\n%s"), _("darktable could not load"), dev->image_storage.filename,
           _("switching to lighttable now."),
-          _("please check the image (use exiv2 or exiftool) for corrupted data. if the image seems to"),
-          _("be intact, please consider opening an issue at https://github.com/darktable-org/darktable.") );
```

Comment:
There is no good reason to split this string into two translatable string. It appears, as it is just because of code formatting.

Reply:
I think the whole string should be something like:
```
"darktable could not load `%s', switching to lighttable now.\n\n"
"please check the image (use exiv2 or exiftool) for corrupted data. if the image seems to\n"
"be intact, please consider opening an issue at https://github.com/darktable-org/darktable."
```
and only format parameter should be `dev->image_storage.filename`.

that way the whole string is translatable and allows for better grammar in languages that would need different construction than "could not load" + "file"

Reply:
Agreed !